src/WilliarinWordpressInteropBundle.php

```diff
@@ -10,7 +10,11 @@
 
 final class WilliarinWordpressInteropBundle extends Bundle
 {
-    public function build(ContainerBuilder $container)
+    /**
+     * @param ContainerBuilder $container
+     * @return void
+     */
```

Comment:
Thanks for your PR! This docblock seems unnecessary, I believe it can be safely removed.